spec/requests/posts_spec.rb

```diff
@@ -1,17 +1,79 @@
 require 'rails_helper'
 
-RSpec.describe 'Posts', type: :request do
-  describe 'GET /index' do
-    it 'returns http success' do
-      get '/posts/index'
-      expect(response).to have_http_status(:success)
+# RSpec.describe 'Posts', type: :request do
+#   describe 'GET #index' do
+#     before(:each) do
+#       get '/users/1/posts'
+#     end
+
+#     it 'When the GET request response status is right' do
+#       expect(response).to have_http_status(:ok)
+#     end
+
+#     it 'When renders the correct template' do
+#       expect(response).to render_template(:index)
+#     end
+
+#     it 'when response  involves right placeholder text' do
+#       expect(response.body).to include('All Posts for')
+#     end
+
+#     describe 'GET #show' do
+#       before(:each) do
+#         get '/users/1/posts/1'
+#       end
+
+#       it 'When the GET request response status is right' do
+#         expect(response).to have_http_status(:ok)
+#       end
+
```

Comment:
* Kindly remove all commented code from your codebase as it makes your codebase bulky and unprofessional 🙏